src/biogeochem/CNCStateUpdate2Mod.F90

```diff
@@ -126,6 +140,12 @@ subroutine CStateUpdate2(num_soilc, filter_soilc, num_soilp, filter_soilp, &
               - cf_veg%m_deadcrootc_xfer_to_litter_patch(p) * dt
          cs_veg%gresp_xfer_patch(p) = cs_veg%gresp_xfer_patch(p)                 &
               - cf_veg%m_gresp_xfer_to_litter_patch(p) * dt
+       else  
+         cs_veg%gresp_storage_patch(p) = cs_veg%gresp_storage_patch(p)           &
+              - cf_veg%m_gresp_storage_to_litter_patch(p) * dt
+         cs_veg%gresp_xfer_patch(p) = cs_veg%gresp_xfer_patch(p)                 &
```

Comment:
This line is exactly identical with lines 141 and 142.
Please take out this line from the if condition and put it right after `p = filter_soilp(fp)` (line 95).

Reply:
done